packages/opentelemetry-instrumentation-langchain/opentelemetry/instrumentation/langchain/utils.py

```diff
@@ -13,11 +13,16 @@ def default(self, o):
             if "callbacks" in o:
                 del o["callbacks"]
                 return o
+            
         if dataclasses.is_dataclass(o):
             return dataclasses.asdict(o)
 
         if hasattr(o, "to_json"):
             return o.to_json()
+        
+        # check if o is a pydantic model
```

Comment:
pydantic should be handled with the `to_json`, no?

Reply:
depends on the version. in pydantic v1 (which is apparently installed in my env) it won't.. they method was throwing an exception which was caught by the `@dont_throw` decorator, took me a while to find it.